test/testEnumerationColoring.cpp

```diff
@@ -0,0 +1,262 @@
+#include <gtest/gtest.h>
```

Comment:
Overall, I'm wondering why we should keep this separate from testVertexColoring.cpp - consider merging

Reply:
Good idea